### PR TITLE
Provide automatic review checklist based on changed paths

### PR DESCRIPTION
**Pull request by okurz**
See
https://github.com/os-autoinst/os-autoinst/pull/1958#issuecomment-1044231841
for an example in os-autoinst.

### CODE REVIEW

File: .github/checklist.yml
Comment (foursixnine) [inline .github/checklist.yml:3]: I like where your heart is at, not sure if it's the  right way though.

**Review by foursixnine — CHANGES_REQUESTED** [review]
Since we're at it, please add one for tests/ for verification runs, we can disable that in the future if it becomes too noisy.

I'd also consider another one for the data directory, asking for a comment on the source

File: .github/checklist.yml
Comment (okurz) [reply .github/checklist.yml:3]: Well, it says "consider", not "you *must*" :)

Comment (foursixnine) [reply .github/checklist.yml:3]: > Well, it says "consider", not "you _must_" :)

That's the le's experiment part

**Comment by okurz** [comment]
> Since we're at it, please add one for tests/ for verification runs, we can disable that in the future if it becomes too noisy.

I did that with https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/14282/files#diff-33df71e78183e3bfe7b5c4ace99dd003696f4472982d2b1eee59af81ac7879d5R6

> I'd also consider another one for the data directory, asking for a comment on the source

Sorry, wasn't sure what you mean there. I have not added that yet. I suggest we do it later. Let's collect feedback with what is suggested for now, ok?

**Comment by foursixnine** [comment]
> > Since we're at it, please add one for tests/ for verification runs, we can disable that in the future if it becomes too noisy.
> 
> I did that with https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/14282/files#diff-33df71e78183e3bfe7b5c4ace99dd003696f4472982d2b1eee59af81ac7879d5R6
> 
> > I'd also consider another one for the data directory, asking for a comment on the source
> 
> Sorry, wasn't sure what you mean there. I have not added that yet. I suggest we do it later. Let's collect feedback with what is suggested for now, ok?

I meant things added in the Data directory, but it isn't something critical anyway :)

**Review by foursixnine — APPROVED** [review]

**Comment by okurz** [comment]
I have to assume that everybody else either gives implicit approval or does not care. So I merge after I have at least received the approval by foursixnine. A more explicit message would be helpful though.

**Pull request merged**
